config-base.js

```diff
@@ -93,7 +101,7 @@ module.exports = {
     'prefer-template': 'warn',
     // disable atomic updates rule until this issue is resolved: https://github.com/eslint/eslint/issues/11899
     'require-atomic-updates': 'off',
-    'require-await': 'off',
+    'require-await': 'off', // use @typescript-eslint/no-floating-promises instead
```

Comment:
I'm not seeing `@typescript-eslint/no-floating-promises` in this list (I see it in overrides)? Should it be here too?

Reply:
this change is from Max's original PR that this is based off of so I can't confirm, but I think it only needs to be in `overrides` and that this comment is just explaining why we have `require-await` set to `off`